include/micrortps/client/client.h

```diff
@@ -29,6 +29,10 @@
 #include <micrortps/client/profile/session/read_access.h>
 #endif //PROFILE_READ_ACCESS
 
+#ifdef PROFILE_WRITE_ACCESS
+#include <micrortps/client/profile/session/write_access.h>
+#endif //PROFILE_READ_ACCESS
```

Comment:
Change `endif` comment to match `ifdef PROFILE_WRITE_ACCESS`.

Reply:
Done!